src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStatic.cs

```diff
@@ -254,6 +255,17 @@ void OnSymbolEnd(SymbolAnalysisContext symbolEndContext)
                 return false;
             }
 
+            // Auto-properties (readonly, get, set) marked with an attribute have a block context callback while they don't
```

Comment:
Can this added code block be moved to an extension method?

Reply:
Sure